SSLP_analyzer/SSLP_App/utils.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+import numpy as np
+import re
+import json
+
+
+def haplo_parser(haplotype):
+    """halpo_parser
+    Parses the haplotype string and extracts the chromosome value and the sslp
+    It does this by extracting the first two ints.
+
+    :param haplotype: haplotype string
+    :return: (str,int,int) haplotype_string, chromosome , sslp
+    """
+    chromosome, sslp = [int(num) for num in re.findall('\d+', haplotype)][0:2]
+    haplo_string = re.sub("^\d+", '', haplotype)
+    return haplo_string,chromosome,sslp
+
+def xslx_parser(filename):
+    """xslx_parser
+    Reads the supplied xslx file and parses it so it can be saved to haplotypes.json.
+    It returns a pandas dataframe which contains the data.
+    :param filename: (str) the name of the file to save it as
+    :return: df (pd.DataFrame) dataframe with all read in data.
+    """
+    df = pd.read_excel(filename,header=None)
```

Comment:
This function is invoked inside a try and except statement. If it does not exists it will throw an exception and notify the user that the file was invalid.